### ISSUE TITLE
Difference in the outcomes of Hugging Face and weights in in ReadMe

### ISSUE THREAD

**Issue by Ashwani-Dangwal** [microsoft/table-transformer#126]
Why is it that the predictions from the Hugging face and predictions from the weights provided in this repo's ReadMe are a bit different.
I am having a hard time trying to understand it. Were there any changes made to the model while importing it to Hugging face?

**Comment by mllife** [comment]
Yes, both implementations are little different, having the same issue. The newer models on GIT TATR are seems to working better.

**Comment by NielsRogge** [comment]
Hi,

With #158 you should get one-on-one results
